android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -931,4 +939,14 @@ public void setFullscreen(boolean fullscreen) {
     public void setUseTextureView(boolean useTextureView) {
         exoPlayerView.setUseTextureView(useTextureView);
     }
+
+    public void setLoadControl(int newMinBufferMs, int newMaxBufferMs, int newBufferForPlaybackMs, int newBufferForPlaybackAfterRebufferMs) {
+        player.release();
```

Comment:
This will generate a null pointer exception if setLoadControl gets called before setSrc. There is no guarantee to the order, it's somewhat random depending on what order you specify props in. I'm not sure if it's a good idea to call `player.release()` without doing all the cleanup that happens in `releasePlayer()` so I would suggest calling `stopPlayback()` which will do the null pointer check and cleanup.

Also, you can remove the `this` since all the variable names are different it doesn't achieve anything.